server/fishtest/templates/base.mak

```diff
@@ -150,11 +152,36 @@
                     title="Upload Neural Network"
                   >
                     <i
-                      class="fa-solid fa-cloud-arrow-up d-inline me-2 mx-lg-1"
+                      class="fa-solid fa-cloud-arrow-up d-inline me-2 px-lg-1"
                     ></i>
                     <span class="d-inline d-lg-none">NN Upload</span>
                   </a>
                 </li>
+                <li class="nav-item col-6 col-lg-auto order-lg-0">
+                  <a
+                    class="nav-link py-2 px-0 px-lg-2"
+                    href="/pull_request"
+                    title="Prepare and submit a pull request"
+                  >
+                    <i
+                      class="fa-solid fa-flag-checkered" id="pull-request-icon" d-inline me-2 mx-lg-1
```

Comment:
The classes are in the wrong place

Reply:
Thanks. I fixed it I think. Html is too tolerant of syntax error.